openmp/runtime/src/kmp_tasking.cpp

```diff
@@ -5452,7 +5454,6 @@ bool __kmpc_omp_has_task_team(kmp_int32 gtid) {
 
 #if OMPX_TASKGRAPH
 // __kmp_find_tdg: identify a TDG through its ID
-// gtid:   Global Thread ID
```

Comment:
Make an NFC change to fix the code comments. Include the other one.

Reply:
same as above